nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/serialization/FlowFromDOMFactory.java

```diff
@@ -148,6 +151,21 @@ public static ReportingTaskDTO getReportingTask(final Element element, final Pro
         return dto;
     }
 
+    public static ParameterProviderDTO getParameterProvider(final Element element, final PropertyEncryptor encryptor, final FlowEncodingVersion flowEncodingVersion) {
+        final ParameterProviderDTO dto = new ParameterProviderDTO();
+
+        dto.setId(getString(element, "id"));
+        dto.setName(getString(element, "name"));
+        dto.setComments(getString(element, "comment"));
```

Comment:
Should these be `comments` instead of `comment`?
```suggestion
        dto.setComments(getString(element, "comments"));
```

Reply:
Interesting.. this looks like an inconsistency between the various components.  For example, Port serializes this as "comments", but Processor serializes it as "comments".  I'll stick with "comment" since this is what Processor, ControllerService, and ReportingTask use.

Reply:
Thanks for checking into this, the inconsistency is unfortunate, but following the convention of other components seems best for now.